src/blog/2023/01/flowforge-1-3-0-released.md

```diff
@@ -0,0 +1,86 @@
+---
+title: FlowForge 1.3 is now available, share your flows through our new team libraries and much more
+subtitle: Our first release of 2023 with some great new features to try out, happy new year from everyone at FlowForge!
+description: 
+date: 2023-01-19 18:00:00.0
+authors: ["rob-marcer"]
+video: ey3xv5j5x7k
+---
+
+Share your flows via team libraries, control access to your Node-RED dashboards using FlowForge credentials, and filter your audit logs by users and actions.
+
+<!--more-->
+
+We're pleased to announce version 1.3 is now available! Due to the recent holiday season, most of our team have been away from their desks but we still have some great new features to share. Keep reading for the details of what's in this release or you can watch our 1 minute roundup video of the new release above.
+
+To make it easy for everyone to experience FlowForge, we are introducing a new [free 30-day trial](https://app.flowforge.com/account/create). With this trial, you can experience the power of using FlowForge to quickly deliver Node-RED applications in a reliable, repeatable, collaborative, and secure manner. To get your trial simply [sign up for a new FlowForge team](https://app.flowforge.com/account/create).
+
+## Features
+
+[Share your flows via team libraries](https://github.com/flowforge/flowforge/issues/237)
```

Comment:
This should be a title so it is on its own line.

Reply:
Fixed for all three main stories in the commit I'm working on now